addons/test_event_full/tests/test_performance.py

```diff
@@ -52,7 +52,7 @@ def test_event_create_batch_notype_website(self):
         batch_size = 20
 
         # simple without type involved + website
-        with freeze_time(self.reference_now), self.assertQueryCount(event_user=3236):  # tef 3173 / com 3172
+        with freeze_time(self.reference_now), self.assertQueryCount(event_user=3569):  # tef 3570 / com 3172
```

Comment:
Wait, what ?

Reply:
`_update_website_menus` calls `website_menu

Reply:
After more investigations, the lang was missing from the context thus `_` was trying very hard to find a lang somewhere to translate some automatic chatter messages, ultimately it used `res.users.context_get()['lang']`. Problem was solved by setting a lang inside the context.